src/main/java/org/jenkinsci/plugins/workflow/actions/ArgumentsAction.java

```diff
@@ -87,7 +85,7 @@ public static boolean isOversized(@CheckForNull Object o, final int maxElements)
         if (maxElements <= 0 ) {
             return false;
         }
-        if (o == null || Primitives.isWrapperType(o.getClass()) || o.getClass().isEnum()) {
+        if (o == null || o.getClass().isPrimitive() || o.getClass().isEnum()) {
```

Comment:
I think these have different meanings. The old Guava code is true for boxed types like `Integer` or `Double`, whereas the new code is true for primitive types like `int` or `double`.

Reply:
good catch